diff --git a/node_modules/@octokit/types/src/GetResponseTypeFromEndpointMethod.ts b/node_modules/@octokit/types/src/GetResponseTypeFromEndpointMethod.ts
index 60ce10c..bab2c37 100644
--- a/node_modules/@octokit/types/src/GetResponseTypeFromEndpointMethod.ts
+++ b/node_modules/@octokit/types/src/GetResponseTypeFromEndpointMethod.ts
@@ -1,5 +1,5 @@
 type Unwrap<T> = T extends Promise<infer U> ? U : T;
-type AnyFunction = (...args) => any;
+type AnyFunction = (...args: any[]) => any;
 
 export type GetResponseTypeFromEndpointMethod<T extends AnyFunction> = Unwrap<
   ReturnType<T>
